### ISSUE TITLE
ezTransXP translator throws UnityException

### ISSUE THREAD

**Issue by nanikit** [bbepis/XUnity.AutoTranslator#160]
## What I did
- Delete generated translations.
- Run a game with ezTransXP translator.

## What I expected

Translated UI elements. And it worked with pre-generated translations.

## What I got

UI elements are not translated. The following is the exception message.

```
UnityException: get_dataPath can only be called from the main thread.
Constructors and field initializers will be executed from the loading thread when loading a scene.
Don't use this function in the constructor or field initializers, instead move initialization code to the Awake or Start function.
  at (wrapper managed-to-native) UnityEngine.Application:get_dataPath ()
  at XUnity.Common.Constants.Paths..cctor () [0x00000] in <filename unknown>:0 
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for XUnity.Common.Constants.Paths

UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
UnityEngine.UnhandledExceptionHandler:<RegisterUECatcher>m__0(Object, UnhandledExceptionEventArgs)
 
(Filename:  Line: -1)
```

**Comment by gravydevsupreme** [comment]
[XUnity.AutoTranslator-BepIn-5x-4.14.0.zip](https://github.com/bbepis/XUnity.AutoTranslator/files/5939092/XUnity.AutoTranslator-BepIn-5x-4.14.0.zip)
[XUnity.AutoTranslator-ReiPatcher-4.14.0.zip](https://github.com/bbepis/XUnity.AutoTranslator/files/5939093/XUnity.AutoTranslator-ReiPatcher-4.14.0.zip)

I think this version should fix it.

**Comment by nanikit** [comment]
I checked that version resolves the exception. Maybe you have a commit to contribute. If that is merged to this repository, feel free to close this.

**Comment by gravydevsupreme** [comment]
Will be included in next release, but source is not checked into `origin/master` yet. Will close issue once that is done.

**Comment by gravydevsupreme** [comment]
Now included in 4.15.0.
